### PR TITLE
WS-1386 added js documentation

### PR DESCRIPTION
**Pull request by markussWs**

### CODE REVIEW

File: docs/API/create-sitemap.code.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const sitemap = `
```
Comment (kristapsws) [inline docs/API/create-sitemap.code.js:1]: use single quote and fix spacings

File: docs/API/download-scraped-data-in-json-format.code.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const scrapingJobId = 500;
+const outputFile = `/tmp/scrapingjob${scrapingJobId}.json`;
+await client.downloadScrapingJobJSON(scrapingJobId, outputFile);
+
+// read data from file with built in JSON reader
+const reader = new JsonReader(outputFile);
+const rows = await reader.toArray();
+rows.forEach((row) => {
```
Comment (kristapsws) [inline docs/API/download-scraped-data-in-json-format.code.js]:
```
		for await (const row of reader.fetchRows()) {
			console.log(`ROW: ${JSON.stringify(row)} \n`);
		}
```

File: docs/API/update-sitemap.code.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+const sitemap = `
+{
```
Comment (kristapsws) [inline docs/API/update-sitemap.code.js:2]: same as for create sitemap

**Review by kristapsws — CHANGES_REQUESTED** [review]

File: docs/API/create-sitemap.code.js (hunk 1 above)
Comment (markussWs) [reply docs/API/create-sitemap.code.js:1]: done

File: docs/API/update-sitemap.code.js (hunk 3 above)
Comment (markussWs) [reply docs/API/update-sitemap.code.js:2]: done

File: docs/API/download-scraped-data-in-json-format.code.js (hunk 2 above)
Comment (markussWs) [reply docs/API/download-scraped-data-in-json-format.code.js]: done

**Review by kristapsws — APPROVED** [review]

**Review by kristapsws — APPROVED** [review]

**Review by kristapsws — APPROVED** [review]

**Comment by martinsbalodis** [comment]
Review: Kristaps -> Einārs -> Mārtiņš

**Review by kristapsws — APPROVED** [review]

File: docs/API/enable-sitemap-scheduler.code.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+const response = await client.enableSitemapScheduler(123, {
+	"cron_minute": "*/10",
```
Comment (einarsozols) [inline docs/API/enable-sitemap-scheduler.code.js]: I think object keys should be without double quotes, for example, Create Scraping Job section it's similar request and example is without double quotes.

**Review by einarsozols — CHANGES_REQUESTED** [review]

File: docs/API/enable-sitemap-scheduler.code.js (hunk 4 above)
Comment (Grundmanis) [reply docs/API/enable-sitemap-scheduler.code.js]: Yes, You are right, missed that

**Review by einarsozols — APPROVED** [review]

**Pull request merged**